lib/vmdb/deprecation.rb

```diff
@@ -23,13 +18,23 @@ class << self
 
     private
```

Comment:
@bdunne Minor, but do we need this `private` line anymore since all the class methods are individually marked private.